csrc/codegen.cpp

```diff
@@ -3190,6 +3190,14 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
   // Reference:
   // https://docs.nvidia.com/cuda/inline-ptx-assembly
   void handle(const kir::Asm* asm_) final {
+    auto getTypeOrIndexType = [](Val* value) {
```

Comment:
super nitpick: Is it snake-case for lambda functions?
```suggestion
    auto get_type_or_index_type = [](Val* value) {
```

Reply:
Seems like `get_type_or_index_type` per https://google.github.io/styleguide/cppguide.html#Formatting_Lambda_Expressions

Reply:
changed